AmazonChimeSDK/AmazonChimeSDK/audiovideo/data/transcript/TranscriptLanguageWithScore.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  TranscriptLanguageWithScore.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// See [Using Amazon Chime SDK live transcription developer guide](https://docs.aws.amazon.com/chime/latest/dg/process-msgs.html) for details about transcription message types and data guidelines
```

Comment:
just confirm this link is appropriate here?

Reply:
updated.